### PR TITLE
network governance updates

### PR DESCRIPTION
**Pull request by smohan-dw**

### CODE REVIEW

File: node/client/src/lib.rs [hunk 1]
```diff
@@ -70,7 +70,6 @@ pub trait RuntimeApiCollection:
 	+ sp_consensus_grandpa::GrandpaApi<Block>
 	+ sp_block_builder::BlockBuilder<Block>
 	+ frame_system_rpc_runtime_api::AccountNonceApi<Block, AccountId, Index>
-	+ pallet_transaction_payment_rpc_runtime_api::TransactionPaymentApi<Block, Balance>
```
Comment (roynrishingha) [inline node/client/src/lib.rs:73]: Curious to know why we're removing `Block` and `Balance`

File: pallets/membership/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+[package]
+name = "pallet-membership"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
```
Comment (roynrishingha) [inline pallets/membership/Cargo.toml:4]: Should we mention Dhiway instead of Parity?

File: pallets/network-membership/src/weights.rs [hunk 3]
```diff
@@ -50,8 +50,9 @@ use core::marker::PhantomData;
 
 /// Weight functions needed for pallet_extrinsic_authorship.
 pub trait WeightInfo {
-	fn add(a: u32, ) -> Weight;
-	fn remove(a: u32, ) -> Weight;
+	fn add( ) -> Weight;
+	fn renew( ) -> Weight;
+	fn revoke( ) -> Weight;
```
Comment (roynrishingha) [inline pallets/network-membership/src/weights.rs:55]: Is `remove` renamed to `revoke` ?

File: node/client/src/lib.rs (hunk 1 above)
Comment (smohan-dw) [reply node/client/src/lib.rs:73]: We are removing neither. We are only removing the transaction payment, as it is not required for a feeless implementation.

File: pallets/membership/Cargo.toml (hunk 2 above)
Comment (smohan-dw) [reply pallets/membership/Cargo.toml:4]: This is a pallet fork with minimum changes from our side. Hence retained the copyright and license.

File: pallets/network-membership/src/weights.rs (hunk 3 above)
Comment (smohan-dw) [reply pallets/network-membership/src/weights.rs:55]: yes, with some signature changes

**Comment by smohan-dw** [comment]
replaced by #223

**Pull request closed without merging**